### ISSUE TITLE
libreoffice crashes when opening any document

### ISSUE THREAD

**Issue by olegkapitonov** [haikuports/haikuports#8587]
OS version: R1/beta4 hrev56578+81

Аfter the latest updates, libreoffice can no longer open any document.
The syslog shows the message

```
KERN: runtime_loader: /boot/system/lib/libxmlsec1.so.1.2.33: Could not resolve symbol 'xmlIOFTPMatch'
```

**Comment by Begasus** [comment]
Probably related to the removed functions in the latest version for libxml2.

**Comment by korli** [comment]
Fixed in 8ca03bc11b3902e9a6867a82bf5e3284efaa321d
